vllm/lora/ops/bgmv_expand_slice.py

```diff
@@ -180,3 +171,8 @@ def bgmv_expand_slice(
         **config,
     )
     return
+
+
+bgmv_expand_slice = torch.library.custom_op("lora::bgmv_expand_slice",
```

Comment:
custom_op is torch 2.4+
Probably worth adding something similar to https://github.com/vllm-project/vllm/pull/7886/files#diff-dab7693bd00a09e22d39aee684a7e419aa358a47c4bd20df33d44f5adf60d304R1114-R1115 in punica.py:16 around the import